### ISSUE TITLE
activities: Include google groups in the teams and update them when changing members

### ISSUE THREAD

**Issue by BrunoRosendo** [NIAEFEUP/website-niaefeup-backend-old#18]
We started using Google Groups to maintain the teams of the projects and events, to use the google calendar integration with it.
It would be very cool if we could use the website to maintain this instead (the interface isn't the nicest) and update the groups automatically.

This requires adding the group "email" address to the project/event and making sure the registered email is the one people use for google calendar (this is more logistics, probably not much to be done in backend)

**Comment by limwa** [comment]
If we automate this, it would be amazing. I sometimes have trouble finding the e-mails of certain members or forget to add/remove them for a few weeks. We could see if Google has an API that let's us work with this!

**Comment by BrunoRosendo** [comment]
We could even think about having a tool to schedule events in the website, so we wouldn't have to keep track of the e-mails (but that's probably for after the MVP is well and done)

**Comment by limwa** [comment]
Yes, I agree, that will take a considerable amount of effort. In the meantime, we can just add the Google Groups e-mails to the descriptions of Slack channels so everyone knows where to find them.

I have been looking into Google's APIs and I have found the [Directory API](https://developers.google.com/admin-sdk/directory) which is what we are looking for. It comes with an SDK that has versions for Java, so that should make things considerably easier.

**Comment by Isaac3105** [comment]
As we are shifting towards building a new backend from zero this issue will be abandoned.
